lib/utils.pm

```diff
@@ -576,11 +576,16 @@ sub zypper_call {
         my $msg = "'zypper -n $command' failed with code $ret";
         if ($ret == 104) {
             $msg .= " (ZYPPER_EXIT_INF_CAP_NOT_FOUND)\n\nRelated zypper logs:\n";
-            script_run('tac /var/log/zypper.log | grep -F -m1 -B10000 "Hi, me zypper" | tac | grep \'\(SolverRequester.cc\|THROW\|CAUGHT\)\' > /tmp/z104.txt');
+            script_run('tac /var/log/zypper.log | grep -F -m1 -B100000 "Hi, me zypper" | tac | grep \'\(SolverRequester.cc\|THROW\|CAUGHT\)\' > /tmp/z104.txt');
```

Comment:
You're right, I did the change because in my new case, 10k lines were not enough for hitting a match and thought it might happen in the other cases too.